scripts/setup-vm-user.sh

```diff
@@ -6,18 +6,21 @@ LOGIN_PASS=$2
 # create the login user if it does not already exist
 if getent passwd $LOGIN_USER >/dev/null; then
   echo "Login User '$LOGIN_USER' already created..."
-else
-  # create user, set password and add it to the usual groups
-  adduser $LOGIN_USER --gecos "" --home "/home/$LOGIN_USER" --disabled-password
-  echo "$LOGIN_USER:$LOGIN_PASS" | chpasswd
-  usermod -a -G adm,cdrom,sudo,dip,plugdev,lpadmin,sambashare $LOGIN_USER
+  exit 0
+fi
 
-  # ensure the new user can do passwordless sudo
-  echo "$LOGIN_USER ALL=(ALL) NOPASSWD: ALL" | sudo tee /etc/sudoers.d/$LOGIN_USER
+# create user, set password and add it to the usual groups
+adduser $LOGIN_USER --gecos "" --home "/home/$LOGIN_USER" --disabled-password
+echo "$LOGIN_USER:$LOGIN_PASS" | chpasswd
+usermod -a -G adm,cdrom,sudo,dip,plugdev,lpadmin,sambashare $LOGIN_USER
 
-  # set the new user as the default in the login screen and lock the current (vagrant user's) screen session
-  if [[ $(which dm-tool) ]]; then
-    echo -e "[SeatDefaults]\nautologin-user=$LOGIN_USER" | sudo tee /etc/lightdm/lightdm.conf
-    XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0 dm-tool lock
-  fi
-fi
+# ensure the new user can do passwordless sudo
+echo "$LOGIN_USER ALL=(ALL) NOPASSWD: ALL" | sudo tee /etc/sudoers.d/$LOGIN_USER
+
+# set the new user as the default in the login screen
+mkdir -p /etc/gdm3
+> /etc/gdm3/custom.conf
```

Comment:
Is this a shorter form of `touch`? I noticed this work in bash, but might confuse zsh users ;)

Reply:
not quite, it clears the file if it exists:

```
fasmat@riddler:~$ echo "test" > test.txt
fasmat@riddler:~$ cat test.txt 
test
fasmat@riddler:~$ > test.txt
fasmat@riddler:~$ cat test.txt 
fasmat@riddler:~$ 
```

Reply:
The goal is to have the script run idem-potently. Touch would cause the following lines to be appended to the file with every subsequent run.

Reply:
Good point, +1 for idempotence. Thanks for elaborating! :)